docs/CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,171 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [v1.10.0](#v1100)
+  - [Downloads for v1.10.0](#downloads-for-v1100)
+  - [What's New](#whats-new)
+    - [Workload Rebalance](#workload-rebalance)
+    - [Free the resource template name from the limitation of label length](#free-the-resource-template-name-from-the-limitation-of-label-length)
+    - [Usability and reliability Enhancements in the production environment](#usability-and-reliability-enhancements-in-the-production-environment)
+  - [Other Notable Changes](#other-notable-changes)
+    - [API Changes](#api-changes)
+    - [Deprecation](#deprecation)
+    - [Bug Fixes](#bug-fixes)
+    - [Security](#security)
+    - [Features & Enhancements](#features--enhancements)
+  - [Other](#other)
+    - [Dependencies](#dependencies)
+    - [Helm Charts](#helm-charts)
+    - [Instrumentation](#instrumentation)
+  - [Contributors](#contributors)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# v1.10.0
+## Downloads for v1.10.0
+
+Download v1.10.0 in the [v1.10.0 release page](https://github.com/karmada-io/karmada/releases/tag/v1.10.0).
+
+## What's New
+
+### Workload Rebalance
+
+This release introduced a workload rebalancing capability. It can actively trigger a brand fresh rescheduling to establish an entirely new replica distribution across clusters. With the workload rebalancing capability, users can trigger a workload rebalancing on demand if the current replicas distribution is not optimal.
+
+See [the documentation](https://karmada.io/docs/next/userguide/scheduling/workload-rebalancer) for more details.
```

Comment:
```suggestion
This release introduced a workload rebalancing capability. It can actively trigger a brand fresh rescheduling to establish 
an entirely new replicas distribution across clusters. When the user has propagated the workloads to member clusters, in some scenarios the current replicas distribution is not the most expected, such as:

* replicas migrated due to cluster failover, but now cluster recovered.
* replicas migrated due to application-level failover, but now each cluster has sufficient resources to run the replicas.
* as for `Aggregated` schedule strategy, replicas were initially distributed across multiple clusters due to resource constraints, but now one cluster is enough to accommodate all replicas.

Thus, with the workload rebalancing capability, users can trigger a workload rebalancing on demand if the current replicas distribution is not optimal.

See [User Guide](https://karmada.io/docs/next/userguide/scheduling/workload-rebalancer) for detail description about this feature and [Tutorial](https://karmada.io/docs/next/tutorials/workload-rebalancer) for specific demo presentation. 
```